.travis.yml

```diff
@@ -10,13 +10,17 @@ python:
   - "3.5"
   - "3.6"
 
+env:
+  - DOCKER_IMAGE=stodevx/cs251-toolkit
+
 install:
   - pip install tox
   - pip install tox-travis
   - pip install coveralls
 
 script:
   - tox
+  - script/docker-build $DOCKER_IMAGE:$TRAVIS_COMMIT
```

Comment:
I'd like to move this somewhere else, possibly just into the deploy `script:` keys?